app/customer.py

```diff
@@ -0,0 +1,121 @@
+import json
+import math
+import os
+from datetime import datetime
+from decimal import Decimal
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+
+    list_customer = []
+
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: Decimal,
+            car: Car
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    @classmethod
+    def create_customer(cls) -> None:
+        Car.create_car()
+        config_path = os.path.join(os.path.dirname(__file__), "config.json")
+        with open(config_path, "r") as file_customer:
+            data = json.load(file_customer)
+
+        for custom in data.get("customers", []):
+            customer = Customer(
+                name=custom.get("name"),
+                product_cart=custom.get("product_cart"),
+                location=custom.get("location"),
+                money=custom.get("money"),
+                car=Car.cars_dict[custom.get("name")],
+            )
+            Customer.list_customer.append(customer)
+
+    def distance_to_shop(self, some_shop: Shop) -> float:
```

Comment:
this would be rather a Car method - not a customer method. it doesn't depend on a customer in any way so it would be logical to move it out. Your customer already has a lot of methods and having one class that does it all is not always a good idea